### PR TITLE
Thoms' changes to non main text

### PR DESCRIPTION
**Pull request by AnthonyLim23**
@thomasholmrod changes to bib file and acknowledgments

### CODE REVIEW

File: bibliography.bib [hunk 1]
```diff
@@ -67,175 +68,211 @@ @article{Kaestner_2011
   journal = {Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment}
 }
 
-@misc{Mantid,
+@misc{mantid,
```
Comment (AnthonyLim23) [inline bibliography.bib]: need to be very careful with this to make sure we dont miss any citations with the old key

Comment (thomasholmrod) [reply bibliography.bib]: There are no missing key message for mantid / Mantid in overleaf.

**Comment by AnthonyLim23** [comment]
@DanNixon we should merge this

**Comment by DanNixon** [comment]
> Note that online was not recognized by the bib style used,

I have a feeling this may be a limitation of whatever toolchain Overleaf uses, using TexLive this worked fine.

**Comment by DanNixon** [comment]
> @DanNixon we should merge this

No, let's wait until the rest of the edits are done. Some of the references will not be needed and others may be.

Really this shouldn't have been a separate PR.

**Comment by DanNixon** [comment]
Sigh... several of these bibliography entries are badly formatted to the point of not working...

**Pull request closed without merging**